[export] Fix torch.export.load with storage offset

yiming0416 opened the pull request:
Summary: As titled

Test Plan:
CI

Rollback Plan:

Differential Revision: D81687701

angelayi reviewed (commented):
thanks for the quick fix! could you add a test?

yiming0416 committed:
[export] Fix torch.export.load with storage offset (#162172)

Summary:

As titled

Test Plan:
CI

Rollback Plan:

Reviewed By: angelayi

Differential Revision: D81687701
